src/main/resources/templates/db_scripts/schema.json

```diff
@@ -3,7 +3,12 @@
     {
       "run": "after",
       "snippetPath": "create_custom_fields_table.sql",
-      "fromModuleVersion": "mod-users-15.7.0"
+      "fromModuleVersion": "19.3.0"
```

Comment:
Can you please tell why you are changing the fromModuleVersion existing script ?

Reply:
Ok got it. Thanks.